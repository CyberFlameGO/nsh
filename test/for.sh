# Break from a nested for & if.
for x in loop1----- loop2----- loop3------
do
    echo $x
    for y in foo bar a b c d e
    do
        if [ $y = a ]
        then
            if [ 1 -eq 1 ]
            then
                break
            fi
        fi

        echo $y
    done
done
