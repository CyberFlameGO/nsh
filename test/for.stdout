loop1-----
foo
bar
loop2-----
foo
bar
loop3------
foo
bar
